### PR TITLE
Networking

### PR DESCRIPTION
**Pull request by aclogar**
Added basic networking fundamentals and a main menu that can be modified to our theme when we have proper assets.  This includes a settings page that can be used to later change game settings.  Currently have a scene loader that can load a scene that is loaded and specified by the name of the scene.

### CODE REVIEW

File: project_bouncy_boi/Assets/Animation/MainMenu/Closed.anim [hunk 1]
```diff
@@ -0,0 +1,148 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!74 &7400000
```
Comment (nayrpg) [inline project_bouncy_boi/Assets/Animation/MainMenu/Closed.anim:3]: What the hell does this even mean!!!

File: project_bouncy_boi/Assets/Animation/Panel/Closing.anim [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    m_Extent: {x: 0, y: 0, z: 0}
+  m_ClipBindingConstant:
+    genericBindings:
+    - path: 0
+      attribute: 2
+      script: {fileID: 0}
+      classID: 4
+      customType: 0
+      isPPtrCurve: 0
+    pptrCurveMapping: []
```
Comment (nayrpg) [inline project_bouncy_boi/Assets/Animation/Panel/Closing.anim:43]: You left your list empty... That is essentially the same as leaving your pants down

File: project_bouncy_boi/Assets/Scripts/DropMe.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+public class DropMe : MonoBehaviour, IDropHandler, IPointerEnterHandler, IPointerExitHandler
+{
+	public Image containerImage;
+	public Image receivingImage;
+	private Color normalColor;
```
Comment (nayrpg) [inline project_bouncy_boi/Assets/Scripts/DropMe.cs:10]: Oh now a color is normal huh?!?!?

File: project_bouncy_boi/Assets/Scripts/DropMe.cs [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	public Color highlightColor = Color.yellow;
```
Comment (nayrpg) [inline project_bouncy_boi/Assets/Scripts/DropMe.cs:11]: Why's it got to be yellow?

File: project_bouncy_boi/Assets/Animation/Panel/Open.anim [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+      m_PostInfinity: 2
+    attribute: m_LocalRotation.z
+    path: 
+    classID: 224
+    script: {fileID: 0}
+  - curve:
+      serializedVersion: 2
+      m_Curve:
+      - time: 0
+        value: -.965925813
```
Comment (nayrpg) [inline project_bouncy_boi/Assets/Animation/Panel/Open.anim:108]: This is as magical of a number as you possibly could get. Did this come from Hogwarts or something?

**Review by nayrpg — APPROVED** [review]

File: project_bouncy_boi/Assets/Animation/Panel/Open.anim (hunk 5 above)
Comment (aclogar) [reply project_bouncy_boi/Assets/Animation/Panel/Open.anim:108]: Narnia actually.

**Pull request merged**